docs/how_to_guides/creating_and_editing_expectations/how_to_create_expectations_that_span_multiple_batches_using_evaluation_parameters.rst

```diff
@@ -0,0 +1,139 @@
+    
+    It's common (but not required) for both Batches to come from the same :ref:`Datasource <Datasources>` and :ref:`BatchKwargsGenerator`.
+
+    .. code-block:: python
+
+        upstream_batch = context.get_batch(
+            context.build_batch_kwargs("my_datasource", "my_generator_name", "my_data_asset_name_1"),
+            expectation_suite_name='my_expectation_suite_1'
+        )
+
+        downstream_batch = context.get_batch(
+            context.build_batch_kwargs("my_datasource", "my_generator_name", "my_data_asset_name_2"),
+            expectation_suite_name='my_expectation_suite_2'
+        )
+
+#. **Define an Expectation using an Evaluation Parameter on the downstream Batch.**
+
+    .. code-block:: python
+
+        eval_param_urn = 'urn:great_expectations:validations:my_expectation_suite_1:expect_table_row_count_to_be_between.result.observed_value'
+        downstream_batch.expect_table_row_count_to_equal(
+            value={
+                '$PARAMETER': eval_param_urn, # this is the actual parameter we're going to use in the validation
+                '$PARAMETER.' + eval_param_urn: 10  # this is a *temporary* value so we can execute the notebook
+            }
+        )
+    
+    The core of this is a ``$PARAMETER : URN`` pair. When Great Expectations encounters a ``$PARAMETER`` flag during validation, it will replace the ``URN`` with a value retrieved from an :ref:`Evaluation Parameter Store` or :ref:`Metrics Store`.
+    
+    This declaration above includes two ``$PARAMETERS``. The first is the real parameter that will be used after the Expectation Suite is stored and deployed in a Validation Operator. The second parameter supports immediate evaluation in the notebook.
+
+    When executed in the notebook, this Expectation will generate an :ref:`Expectation Validation Result`:
+
+    .. code-block:: python
+
+        {
+            "result": {
+                "observed_value": 506
+            },
+            "meta": {},
+            "exception_info": null,
+            "success": false
+        }
+
+    .. warning::
+
+        Your URN must be exactly correct in order to work in production. Unfortunately, successful execution at this stage does not guarantee that the URN is specified correctly and that the intended parameters will be available when executed later.
+        
+#. **Save your Expectation Suite**
+
+    .. code-block:: python
+
+        downstream_batch.save_expectation_suite(discard_failed_expectations=False)
+
+    This step is necessary, because your ``$PARAMETER`` will only function properly when invoked within a Validation operation with multiple Batches, the simplest way to execute such an operation invoke is through a :ref:`Validation Operator`, and Validation Operators are configured to load Expectation Suites from Expectation Stores, not memory.
```

Comment:
```suggestion
    This step is necessary because your ``$PARAMETER`` will only function properly when invoked within a Validation operation with multiple Batches. The simplest way to execute such an operation is through a :ref:`Validation Operator`, and Validation Operators are configured to load Expectation Suites from Expectation Stores, not memory.
```

Note: this isn't strictly correct. The Validation Operator could be provided batches that already have suites attached to them.

Reply:
I'm confused. That's what I tried first and it didn't work.

Specifically, if you run all the same steps, minus the `save_expectation_suite` step, the V.O. will not execute the new Expectation. Maybe the new Expectation is being discarded before execution, because it failed evaluation earlier...?